packet/k8s_client.go

```diff
@@ -0,0 +1,100 @@
+package packet
+
+import (
+	"context"
+
+	"github.com/packethost/pkg/log"
+	"github.com/pkg/errors"
+	"github.com/tinkerbell/tink/pkg/apis/core/v1alpha1"
+	"github.com/tinkerbell/tink/pkg/controllers"
+	"github.com/tinkerbell/tink/pkg/convert"
+	"github.com/tinkerbell/tink/protos/workflow"
+	"google.golang.org/grpc"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type K8sClient struct {
```

Comment:
what do you think about changing `K8sClient` to `K8SClient`? 

_"Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case."_ -- [ref](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
K8s isn't so much an acronym as a shorthand